nautobot_chatops/dispatchers/ms_teams.py

```diff
@@ -229,3 +229,7 @@ def user_mention(self):
         #   channel-and-group-conversations?tabs=python#adding-mentions-to-your-messages
         # For now we just use the user name as plaintext, not an actual mention
         return f"{self.context['user_name']}"
+
+    def bold(self, text):
+        """Mark text as bold."""
+        return f"*{text}*"
```

Comment:
Does this fix #93?

Reply:
Ugg, looks like I have a collision in my branches!!!! I have a separate branch that I was preparing for that.

Reply:
Yes it does. I will add that into here.